controllers/wavefront_controller_test.go

```diff
@@ -245,26 +231,11 @@ func TestReconcileCollector(t *testing.T) {
 		))
 	})
 
-	t.Run("defaults values for default collector config", func(t *testing.T) {
```

Comment:
This test wasn't actually testing any actual defaults. It was testing the test setup defaults, so basically this test was testing nothing.